app/assets/stylesheets/layouts/_header.css.scss

```diff
@@ -1,4 +1,28 @@
-.header {
-  &.top-bar { background: $header-green !important; }
+nav.header {
+  background: $header-green !important;
   li, a { background: $header-green !important; }
 }
+
+.magellan.header {
+  background: $header-green !important;
+
+  ul, li { margin: 0; }
+  li, a { background: $header-green !important; }
```

Comment:
!important should not be used<br>Each selector in a comma sequence should be on its own line